deploy/charts/harvester/templates/cattle-system.yaml

```diff
@@ -19,6 +19,37 @@ spec:
   upgrade:
     image: {{ .Values.generalJob.image.repository }}:{{ .Values.generalJob.image.tag }}
     command: ["/bin/sh", "-c"]
+    # We can't use systemctl in docker image.
+    # ref: https://stackoverflow.com/questions/49285658/how-to-solve-docker-issue-failed-to-connect-to-bus-no-such-file-or-directory
     args:
-      - cp /ca/additional-ca.pem /host/etc/pki/trust/anchors;
-      - chroot /host update-ca-certificates
+      - echo "Update additional ca ...";
+        cp /ca/additional-ca.pem /host/etc/pki/trust/anchors;
+        chroot /host update-ca-certificates;
+        echo "Restart RKE2 ...";
+        chroot /host /bin/bash -c "kill $(pgrep rke2);"
+        echo "Done";
+---
+apiVersion: upgrade.cattle.io/v1
+kind: Plan
+metadata:
+  name: sync-rke2-registries
+  namespace: cattle-system
+spec:
+  concurrency: 1
+  nodeSelector:
+    matchLabels:
+      harvesterhci.io/managed: "true"
+  serviceAccountName: system-upgrade-controller
+  secrets:
+    - name: harvester-rke2-registries
+      path: /registries
+  version: v1.0.2
```

Comment:
do we use this to track our versions? if so probably this should be `v1.1.0`?

Reply:
Updated to `v1.1.0`. Thank you.